objectstore-server/src/config.rs

```diff
@@ -31,7 +31,8 @@ pub struct Config {
     pub http_addr: SocketAddr,
 
     // storage config
-    pub storage: Storage,
+    pub small_storage: Storage,
+    pub large_storage: Storage,
```

Comment:
Can we give these storages different designation than "small" and "large" in config? I think this conveys wrong semantics. 

From our cost modeling, we've seen that the two primary reasons we need two backends are:
- Retention time: With long retention (release files and debug files), resident data becomes the cost driver, prompting a backend with cold storage options.
- QPS: With high operation throughput, the per-ops cost becomes the cost driver, prompting a backend with low or no per-ops cost, at the expense of more costly storage.

The file size is an initially sufficient heuristic to make the decision of which backend to use, but we should decouple this from how the backends are identified. I can quickly think of "high_volume" and "long_term", what would you suggest?